thoth/package_extract/core.py

```diff
@@ -69,10 +69,11 @@ def extract_image(image_name: str, timeout: int = None, *,
         result = run_analyzers(rootfs_path)
         result['layers'] = layers
 
-    push_gateway = os.getenv('PROMETHEUS_PUSHGATEWAY_HOST','PROMETHEUS_PUSHGATEWAY_HOST:PROMETHEUS_PUSHGATEWAY_PORT')
-    if push_gateway:
+    push_gateway_host = os.getenv('PROMETHEUS_PUSHGATEWAY_HOST')
```

Comment:
what if the environment variable is not set?

Reply:
Then we'll not push any metrics. Treat it as a switch.

Reply:
Then we'll not push any metrics. Treat it as a switch.

Reply:
> Then we'll not push any metrics.
I guess we will have a problem bit pushadd() as it tries to push to a host called "host:"